### PR TITLE
Instructor: view results: remove 'filter results' feature #7295

### LINKED ISSUE #7295: Instructor: view results: remove 'filter results' feature
Current:
image

Suggestion:
Remove the filter box

Justification:
It simply filters the question text, so not that useful. We can remove this superficial 'filter' feature until we have a 'filter' feature with more substance

### PR DESCRIPTION
**Pull request by LimeTheCoder**
Fixes #7295 
View results after removing filter results:
image

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@LimeTheCoder do not forget to remove the back-end functions as well. If you remove only the front-end (which is the case currently), the back-end codes will be dead codes. Of course, make sure that they are not used anywhere else first.

**Comment by amarlearning** [comment]
@LimeTheCoder Remember to run Godmode on failing test.

File: src/main/webapp/js/instructorFeedbackResults.es6 [hunk 1]
```diff
@@ -140,11 +140,6 @@ function filterResults(rawSearchText) {
     }
 }
 
-function updateResultsFilter() {
-    $('input[id=filterTextForDownload]').val($('#results-search-box').val());
-    filterResults($('#results-search-box').val());
```
Comment (wkurniawan07) [inline src/main/webapp/js/instructorFeedbackResults.es6]: Looks like you can remove this function as well.

Comment (LimeTheCoder) [inline src/main/webapp/js/instructorFeedbackResults.es6]: Already removed as part of that commit: [https://github.com/TEAMMATES/teammates/pull/7313/commits/7432c87770038f91605e35489b04edb001680d23#diff-8582992252efae5107e2b40ec07b54a6](https://github.com/TEAMMATES/teammates/pull/7313/commits/7432c87770038f91605e35489b04edb001680d23#diff-8582992252efae5107e2b40ec07b54a6l)

Comment (wkurniawan07) [inline src/main/webapp/js/instructorFeedbackResults.es6]: Oops sorry for not being clear. What I meant was the function named `filterResults`.

**Comment by whipermr5** [comment]
@LimeTheCoder There are new merge conflicts 😅

**Pull request merged**